docs/installation/mac.md

```diff
@@ -76,4 +80,38 @@ LoadError: dlopen(/Users/<username>/.rbenv/versions/2.6.3/lib/ruby/2.6.0/x86_64-
 
 **_Solution:_** Run `ln -s /usr/local/opt/readline/lib/libreadline.dylib /usr/local/opt/readline/lib/libreadline.<some version number>.dylib` from the command line then run `bin/setup` again. You may have a different version or libreadline, so replace `<some version number>` with the version that errored.
 
+---
+
+**Error:**
+
+```shell
+PG::Error: ERROR:  invalid value for parameter "TimeZone": "UTC"
+: SET time zone 'UTC'
+```
+
+**_Solution:_** Restart your Postgres.app, or, if you installed PostgreSQL with Homebrew, restart with:
+
+```shell
+pg_ctl -D /usr/local/var/postgres stop
```

Comment:
The easiest way to restart from Homebrew is `brew services restart postgresql`

Reply:
Changed, thanks for the quick review!